### PR TITLE
CLA on Amoy

### PR DESCRIPTION
**Pull request by thedriftofwords**
https://documentation-git-fork-thedriftofwords-aut-c97e08-chainlinklabs.vercel.app/chainlink-automation/overview/automation-release-notes

https://documentation-git-fork-thedriftofwords-aut-c97e08-chainlinklabs.vercel.app/chainlink-automation/overview/supported-networks#amoy-testnet

### CODE REVIEW

File: src/features/chainlink-automation/data/chainlink-automation-config.json [hunk 1]
```diff
@@ -124,6 +124,31 @@
     "registrar": "0x0Bc5EDC7219D272d9dEDd919CE2b4726129AC02B",
     "transcoder": "0x0000000000000000000000000000000000000000"
   },
+  "POLYGON_AMOY": {
+    "paymentPremiumPPB": 700000000,
+    "blockCountPerTurn": "Not Applicable",
+    "maxCheckDataSize": 5000,
+    "checkGasLimit": 6500000,
```
Comment (friedemannf) [inline src/features/chainlink-automation/data/chainlink-automation-config.json:131]:
```suggestion
    "checkGasLimit": 10000000,
```

File: src/features/chainlink-automation/data/chainlink-automation-config.json [hunk 2]
```diff
@@ -124,6 +124,31 @@
...
+    "gasCeilingMultiplier": 3,
+    "minUpkeepSpend": {
+      "type": "BigNumber",
+      "hex": "0x00"
+    },
+    "maxPerformGas": 5000000,
+    "fallbackGasPrice": {
+      "type": "BigNumber",
+      "hex": "0x2e90edd000"
```
Comment (friedemannf) [inline src/features/chainlink-automation/data/chainlink-automation-config.json:140]:
```suggestion
      "hex": "0xe8d4a51000"
```

File: src/features/chainlink-automation/data/chainlink-automation-config.json [hunk 3]
```diff
@@ -124,6 +124,31 @@
...
+    },
+    "fallbackLinkPrice": {
+      "type": "BigNumber",
+      "hex": "0x470de4df820000"
```
Comment (friedemannf) [inline src/features/chainlink-automation/data/chainlink-automation-config.json:144]:
```suggestion
      "hex": "0x5c810d82e2bcd360"
```

File: src/features/chainlink-automation/data/chainlink-automation-config.json [hunk 4]
```diff
@@ -124,6 +124,31 @@
...
+    },
+    "maxPerformDataSize": 5000,
+    "flatFeeMicroLink": 10000,
+    "stalenessSeconds": 90000,
```
Comment (friedemannf) [inline src/features/chainlink-automation/data/chainlink-automation-config.json:148]:
```suggestion
    "stalenessSeconds": 360000,
```

File: src/features/chainlink-automation/data/chainlink-automation-config.json [hunk 5]
```diff
@@ -124,6 +124,31 @@
...
+    "registrar": "0x99083A4bb154B0a3EC7a0D1eb40370C892Db4225",
+    "transcoder": "0x0000000000000000000000000000000000000000"
```
Comment (friedemannf) [inline src/features/chainlink-automation/data/chainlink-automation-config.json:150]:
```suggestion
    "transcoder": "0x703C1d261a996755409c74d00871e7D6Af4d9896"
```

**Review by friedemannf — CHANGES_REQUESTED** [review]
Also, could we use `https://amoy.polygonscan.com/` as the explorer?

**Comment by thedriftofwords** [comment]
> Also, could we use `https://amoy.polygonscan.com/` as the explorer?

Yes I have that in #1904

**Review by dwightjl — APPROVED** [review]

**Comment by thedriftofwords** [comment]
https://automation.chain.link/polygon-amoy is live

**Pull request merged**